TwitchChannelPointsMiner/classes/Twitch.py

```diff
@@ -244,6 +262,35 @@ def post_gql_request(self, json_data):
             )
             return {}
 
+    # Request for Integrity Token
+    # Twitch needs Authorization, Client-Id, X-Device-Id to generate JWT which is used for authorize gql requests
+    def post_integrity(self):
+        if (
+            datetime.now().timestamp() * 1000 - self.integrity_expire < 0
```

Comment:
It would be better to add a slight buffer to this (e.g. anything less than 5 minutes before expiry)

Reply:
- Add 5 minutes buffer to regenerate `Integrity Token` before expire [186ea09](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/pull/594/commits/186ea09e042b991c1f674df0c884fe6976cf2590)